Classes/TreeTab.lua

```diff
@@ -65,34 +75,48 @@ local TreeTabClass = newClass("TreeTab", "ControlHost", function(self, build)
+	self.controls.reset = new("ButtonControl", {"LEFT",self.controls.specSelect,"RIGHT"}, 8, 0, 60*SW, 20, "Reset", function()
 		main:OpenConfirmPopup("Reset Tree", "Are you sure you want to reset your passive tree?", "Reset", function()
 			self.build.spec:ResetNodes()
 			self.build.spec:AddUndoState()
 			self.build.buildFlag = true
 		end)
 	end)
-	self.controls.import = new("ButtonControl", {"LEFT",self.controls.reset,"RIGHT"}, 8, 0, 90, 20, "Import Tree", function()
+	self.controls.import = new("ButtonControl", {"LEFT",self.controls.reset,"RIGHT"}, 8, 0, 90*SW, 20, "Import Tree", function()
 		self:OpenImportPopup()
 	end)
-	self.controls.export = new("ButtonControl", {"LEFT",self.controls.import,"RIGHT"}, 8, 0, 90, 20, "Export Tree", function()
+	self.controls.export = new("ButtonControl", {"LEFT",self.controls.import,"RIGHT"}, 8, 0, 90*SW, 20, "Export Tree", function()
 		self:OpenExportPopup()
 	end)
-	self.controls.treeSearch = new("EditControl", {"LEFT",self.controls.export,"RIGHT"}, 8, 0, 300, 20, "", "Search", "%c%(%)", 100, function(buf)
+	self.controls.treeSearch = new("EditControl", {"LEFT",self.controls.export,"RIGHT"}, 8, 0, 300*SW, 20, "", "Search", "%c%(%)", 100, function(buf)
 		self.viewer.searchStr = buf
-	end)
-	self.controls.treeHeatMap = new("CheckBoxControl", {"LEFT",self.controls.treeSearch,"RIGHT"}, 130, 0, 20, "Show Node Power:", function(state)	
+	end);
+	self.controls.treeHeatMap = new("CheckBoxControl", {"LEFT",self.controls.treeSearch,"RIGHT"}, 130, 0, 20, "Show Node Power:", function(state)
 		self.viewer.showHeatMap = state
 	end)
-	self.controls.treeHeatMapStatSelect = new("DropDownControl", {"LEFT",self.controls.treeHeatMap,"RIGHT"}, 8, 0, 150, 20, nil, function(index, value)
+	self.controls.treeHeatMapStatSelect = new("DropDownControl", {"LEFT",self.controls.treeHeatMap,"RIGHT"}, 8, 0, 150*SW, 20, nil, function(index, value)
 		self:SetPowerCalc(value)
 	end)
+	local dimX = 452
+	local dimY = 500
+	self.controls.powerList = new("TextListControl", nil, (ScreenW-dimX), (ScreenH/2-dimY/2), dimX, dimY, nil, {})
+	self.controls.powerList.shown = false
+	self.controls.listPower = new("ButtonControl", {"LEFT",self.controls.treeHeatMapStatSelect,"RIGHT"}, 8, 0, 72*SW, 20, "Node List", function()
+		self:updatePower()
+		if (self.controls.powerList:IsShown()) then
+			self.controls.powerList.shown = false
+		else
+			self.controls.powerList.shown = true
+		end
+	end)
+	self.controls.listPower.tooltipText = function()
+		return "Show / hide node power list.\nContains a list of nodes sorted by positive change in selected highlight stat"
+	end
 	self.controls.treeHeatMap.tooltipText = function()
 		local offCol, defCol = main.nodePowerTheme:match("(%a+)/(%a+)")
 		return "When enabled, an estimate of the offensive and defensive strength of\neach unallocated passive is calculated and displayed visually.\nOffensive power shows as "..offCol:lower()..", defensive power as "..defCol:lower().."."
```

Comment:
Is this comment still correct?

Reply:
Yes. It is still the correct description for what Node List does